akka-persistence-query/src/main/scala/akka/persistence/query/scaladsl/CurrentEventsByTagQuery.scala

```diff
@@ -15,8 +15,11 @@ trait CurrentEventsByTagQuery extends ReadJournal {
 
   /**
    * Same type of query as [[EventsByTagQuery#eventsByTag]] but the event stream
-   * is completed immediately when it reaches the end of the "result set". Events that are
-   * stored after the query is completed are not included in the event stream.
+   * is completed immediately when it reaches the end of the "result set". Depending
+   * on journal implementation, this may mean all events up to when the query is
+   * started, or it may include events that are persisted while the query is still
+   * streaming results. For eventually consistent stores, it may only include all
+   * events up to some point before the query is started.
```

Comment:
_Some_ or _specific_ point?

Reply:
I'd say some point - an eventually consistent store could have nodes that are days or even weeks behind others. The point of this statement is that if I were to submit a command to an actor, and it emitted an event, and I got an acknowledgement, there is no guarantee that that event will be returned by this query. I'm pretty sure that for most event stores it will be, but I don't think Akka persistence should require causal ordering of this query with other events. There's no specific guarantee here.